tests/conftest.py

```diff
@@ -129,8 +131,9 @@ def election_settings(client: FlaskClient, election_id: str) -> None:
 
 @pytest.fixture
 def manifests(client: FlaskClient, election_id: str, jurisdiction_ids: List[str]):
+    set_logged_in_user(client, UserType.JURISDICTION_ADMIN, DEFAULT_JA_EMAIL)
     rv = client.put(
-        f"/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/manifest",
+        f"/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/ballot-manifest",
```

Comment:
I'm seeing a lot of these changes from `manifest` to `ballot-manifest` in the tests... but are you still testing the old endpoints for the single-jurisdiction flow? For the sake of high-level review, I trust that you will double check that, and otherwise I'm good :)

Reply:
The only places I am changing tests from the old to the new endpoint is in tests for the multi-jurisdiction flow - tests I've written since I started working on it. The existing tests for the single-jurisdiction flow still use the old endpoint. Just double checked to make sure.